api/urls.py

```diff
@@ -0,0 +1,14 @@
+from rest_framework import routers
+from syllabus.api import CourseViewSet
+from visitor import api as visitor_api
+from bootcamp.api import CompanyViewSet, MentorViewSet, ContentViewSet
+
+router = routers.DefaultRouter()
+router.register('api/courses', CourseViewSet, 'courses')
+router.register('api/students', visitor_api.StudentViewSet, 'students')
+router.register('api/company', CompanyViewSet, 'company')
+router.register('api/mentors', MentorViewSet, 'mentors')
+router.register('api/contents', ContentViewSet, 'contents')
+router.register('api/talk-to-advisor', visitor_api.TalkViewSet, 'talk')
```

Comment:
can version of the API be set?
might you can refer: https://www.django-rest-framework.org/api-guide/versioning/

Reply:
Also, better to namespace rather than adding prefix `api` to all routes.

Reply:
@shakyasaijal did you forget to change above comments

Reply:
@khadkarajesh  dai, nope. Working on it.

Reply:
@shreeramneupane  dai, @khadkarajesh  dai, Done